game/src/js/scenes/GameScene.js

```diff
@@ -78,9 +84,17 @@ export default class GameScene extends BaseScene {
     if(this.localCharacter) {
       const vector = this.controller.getWASDVector();
       if(this.localCharacter.motionChanged(vector)) {
-        console.log('motion changed locally');
         this.localCharacter.setMotion(vector);
-        this.socket.emit('setMotion', this.localCharacter.x, this.localCharacter.y, vector.x, vector.y);
+      }
+
+      if (this.localCharacter.isPositionDifferent(this.lastPosition.x, this.lastPosition.y, vector) &&
```

Comment:
I think orientation is important in case we later want to do something like adding spells that prevent characters from moving, but can still turn, and a hit in the back deals more damage than a hit in the front.